src/components/v2v3/V2V3Project/V2V3PayButton/V2V3ConfirmPayModal/NftRewardCell.tsx

```diff
@@ -1,24 +1,24 @@
 import { LoadingOutlined } from '@ant-design/icons'
-import { Image, Tooltip } from 'antd'
+import { Image, Space, Tooltip } from 'antd'
 import ExternalLink from 'components/ExternalLink'
 
 import { NftRewardTier } from 'models/nftRewardTier'
 import { useState } from 'react'
 import { ipfsToHttps } from 'utils/ipfs'
 
 export function NftRewardCell({
-  nftReward,
+  nftRewards,
 }: {
-  nftReward: NftRewardTier
+  nftRewards: NftRewardTier[]
 }): JSX.Element {
   const [imageLoading, setImageLoading] = useState<boolean>(true)
 
-  const nftImage = (
+  const nftImage = (tier: NftRewardTier) => (
```

Comment:
**suggestion** perhaps this can be its own component, declared outside of the scope of NftRewardCell (can probably still be in this file)

Reply:
Yeah good call